packages/paymasters/contracts/helpers/UniswapV3Helper.sol

```diff
@@ -6,14 +6,15 @@ import "@uniswap/v3-periphery/contracts/interfaces/ISwapRouter.sol";
 import "@uniswap/v3-periphery/contracts/interfaces/IPeripheryPayments.sol";
 
 library UniswapV3Helper {
+    event UniswapReverted(address tokenIn, address tokenOut, uint256 amountIn, uint256 amountOutMin);
     // turn ERC-20 tokens into native unwrapped ETH at market price
-    function swapToEth(
-        address weth,
+    function swapToWeth(
```

Comment:
The comment here says 'native unwrapped' - should say 'wrapped'

Reply:
Fixed